jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/JibPluginTest.java

```diff
@@ -148,4 +150,77 @@ public void testProjectDependencyAssembleTasksAreRun() {
                   .collect(Collectors.toSet()));
         });
   }
+
+  @Test
+  public void testWebappProject() {
+    Project rootProject =
+        ProjectBuilder.builder().withProjectDir(testProjectRoot.getRoot()).withName("root").build();
+    rootProject.getPluginManager().apply("java");
+    rootProject.getPluginManager().apply("war");
+    rootProject.getPluginManager().apply("com.google.cloud.tools.jib");
+    ((ProjectInternal) rootProject).evaluate();
+    Assert.assertNotNull(rootProject.getTasks().getByPath(":" + JibPlugin.EXPLODED_WAR_TASK_NAME));
+    ExplodedWarTask explodedWarTask =
+        (ExplodedWarTask) rootProject.getTasks().getByPath(":" + JibPlugin.EXPLODED_WAR_TASK_NAME);
+    Assert.assertEquals(
+        rootProject
+            .getBuildDir()
+            .toPath()
+            .resolve(ProjectProperties.EXPLODED_WAR_DIRECTORY_NAME)
+            .toFile(),
+        explodedWarTask.getExplodedWarDirectory());
+
+    Assert.assertEquals(
+        explodedWarTask,
+        rootProject
+            .getTasks()
+            .getByPath(JibPlugin.BUILD_IMAGE_TASK_NAME)
+            .getDependsOn()
+            .iterator()
+            .next());
+    Assert.assertEquals(
+        explodedWarTask,
+        rootProject
+            .getTasks()
+            .getByPath(JibPlugin.BUILD_DOCKER_TASK_NAME)
+            .getDependsOn()
+            .iterator()
+            .next());
+    Assert.assertEquals(
+        explodedWarTask,
+        rootProject
+            .getTasks()
+            .getByPath(JibPlugin.BUILD_TAR_TASK_NAME)
+            .getDependsOn()
+            .iterator()
+            .next());
+    Assert.assertEquals(
+        explodedWarTask,
+        rootProject
+            .getTasks()
+            .getByPath(JibPlugin.DOCKER_CONTEXT_TASK_NAME)
+            .getDependsOn()
+            .iterator()
+            .next());
+    Assert.assertEquals(
+        JibPlugin.DEFAULT_WEBAPP_FROM_IMAGE,
+        ((BuildImageTask) rootProject.getTasks().getByPath(JibPlugin.BUILD_IMAGE_TASK_NAME))
+            .getJib()
+            .getBaseImage());
+  }
+
+  @Test(expected = UnknownTaskException.class)
```

Comment:
We don't use `expected` in this codebase. Please follow the practice below:

```java
try {
   theVeryMethodThatThrowsAnException(); // should be narrowly scoped.
   Assert.fail();
} catch (UnknownTaskException ex) {
   Assert.assertNotNull(ex.getMessage);
}
```

Reply:
Done